main.py

```diff
@@ -107,6 +121,30 @@ def get_name_tag(resource_type, name):
 )
 
 
+# Initialize the ELBv2 (ALB) client
+elbv2 = boto3.client('elbv2')
```

Comment:
We can still remove these parts (ALB security group) since it belongs to the ALB PR (#19), otherwise looks good.